src/templates/PoolV2/Activity/index.tsx

```diff
@@ -130,11 +132,14 @@ const Activity = () => {
       }
     }
 
-    const activitiesInvestors = getActivityInfo(
-      _activities,
-      data.pages[0]?.underlying_assets || [],
-      filters
-    )
+    const assets: Record<string, string> = {}
+    tokenListSwapProvider.forEach(token => {
```

Comment:
You decide if you want to change this, but I would use `for of` in here just because it looks good to be honest.